src/app/app.functions/crm-card.js

```diff
@@ -1,23 +1,24 @@
 const axios = require("axios");
 
 exports.main = async (context = {}, sendResponse) => {
-  const fallbackQuote = `I think I do myself a disservice by comparing myself to Steve Jobs and Walt Disney and human beings that we've seen before. It should be more like Willy Wonka...and welcome to my chocolate factory.`;
   const demoObject = {
     objectId: 1,
     title: "Sample project custom CRM card",
     desc: `A custom CRM card is a UI extension that displays custom info from either HubSpot or an external system. It can also include custom actions—click the "Get Inspired" button to see example data from kanye.rest.`,
   };
 
   try {
-    const { data } = await axios.get("https://api.kanye.rest/");
+    const { data } = await axios.get("https://zenquotes.io/api/random");
 
     sendResponse({
       results: [
         demoObject,
         {
           objectId: 2,
-          title: "Wisdom from Kanye",
-          quote: data.quote || fallbackQuote,
+          link: "https://zenquotes.io/",
```

Comment:
I wonder if there is a better way to include a link than this?

Reply:
According to their docs I think that this is sufficient.  " API Usage Limits and Attribution...We require that you show attribution with a link back to https://zenquotes.io/ when using the free version of this API.